<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MantisViewerIHMQtClass</class>
 <widget class="QMainWindow" name="MantisViewerIHMQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MantisViewerIHMQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Onyx</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mon Mantis a moi que j'ai fait toute seule avec mes petites mains</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Projet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_Projet"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_Version"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mantisviewerihmqt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBox_Projet</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MantisViewerIHMQtClass</receiver>
   <slot>onModificationProjet(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onModificationProjet(QString)</slot>
 </slots>
</ui>
